packages/transformers/js/src/JSTransformer.js

```diff
@@ -82,7 +82,15 @@ export default new Transformer({
       walk.ancestor(ast.program, collectDependencies, asset);
     }
 
-    if (!asset.env.isNode()) {
+    if (asset.env.isNode()) {
```

Comment:
@devongovett does `isNode` cover electron too? Maybe safer to just always do this?

Reply:
yeah it does cover electron